pkg/networkmanager/firewall.jsx

```diff
@@ -352,8 +352,11 @@ class AddEditServicesModal extends React.Component {
     }
 
     getCustomId() {
-        const all_ports = this.state.custom_tcp_ports.concat(this.state.custom_udp_ports);
-        return "custom--" + all_ports.map(this.getName).join('-');
+        return "custom--" + (
+            this.state.custom_tcp_ports.map(port => this.getName(port, "tcp"))
+                    .concat(this.state.custom_udp_ports.map(port => this.getName(port, "udp")))
+                    .join('-')
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18490-20230503-105640-9f8cc789-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)